debian/changelog

```diff
@@ -1,3 +1,9 @@
+libpam-usb (0.7.4) unstable; urgency=high
+
+  * Massive rework of local check / deny_remote
```

Comment:
needs more details, also has other fixes missing. best to remove and bump version in a separate PR